examples/complete/main.tf

```diff
@@ -1,7 +1,46 @@
+module "vpc" {
+  source  = "cloudposse/vpc/aws"
+  version = "1.1.0"
+
+  namespace  = var.namespace
+  stage      = var.stage
+  name       = var.name
+  cidr_block = "172.16.0.0/16"
```

Comment:
these inputs will need to be updated

Reply:
@goruha the interface has changed to these modules. 

https://github.com/cloudposse/terraform-aws-vpc

`cidr_block` is now `ipv4_primary_cidr_block`

Other inptus have also changed for vpc and subnets

Reply:
@goruha

Reply:
I use `1.2.0` version that is stable, not a release candidate.
`cidr_block` is still there.